.github/CODEOWNERS

```diff
@@ -1 +1,40 @@
+# File extension owners
+
+*.sol @yorhodes @asaj @tkporter
+*.ts @yorhodes @jmrossy @nambrot
+*.rs @mattiecnvr @tkporter @asaj
+*.md @Skunkchain @asaj
+
+# Package owners
+
+## Contracts
+
+solidity/ @yorhodes @asaj @tkporter
+solidity/contracts/igps @tkporter
+solidity/contracts/isms @asaj
+solidity/contracts/middleware @yorhodes
+
+## Agents
+
+rust/ @mattiecnvr @tkporter @asaj
+rust/agents/relayer @mattiecnvr
```

Comment:
Not sure we need to be this granular?

Reply:
+1, slight preference for "package" level owners, e.g.:

`rust`, `sdk`, `solidity`